### ISSUE TITLE
[10.0] web_widget_image_webcam Error getting stream

### ISSUE THREAD

**Issue by ngrandjean** [OCA/web#1149]
Hello,

Module web_widget_image_webcam worked just fine until very recently. Now I have en error everytime I try to connect to the webcam : 
![image](https://user-images.githubusercontent.com/25527410/50765032-f0626f80-1274-11e9-9227-4ab0fce7f80a.png)

I've tried with Chrome and Firefox.

**Comment by pedrobaeza** [comment]
Have you installed any additional extension? Or maybe have you updated the browser?

**Comment by ngrandjean** [comment]
I guess the browser has been updated yes. After some researches it seems that the method createObjectURL() is deprecated for MediaStream objects (see https://developer.mozilla.org/en-US/docs/Web/API/URL/createObjectURL).

I did a quick fix for my system because it was very blocking but I didn't fully tested it for previous Chrome/Firefox version or for other browsers. Would it still help if I did a PR with it ?
